projects/resupply/index.js

```diff
@@ -0,0 +1,47 @@
+const sdk = require('@defillama/sdk')
+const pairContract = require('./abi/pairContract')
+const { get } = require('../helper/http');
+
+// getAllPairAddresses-- 0x607b6d16
+// After for each of the pairs, we do the following:
+// Call totalCollateral-- 0x4ac8eb5f for each
+// 
+
+const BLOCK_START = 22034910;
+
+async function tvl(api) {
+    const pairs = await get('https://raw.githubusercontent.com/resupplyfi/resupply/main/deployment/contracts.json');
```

Comment:
change it to use `getConfig` it caches the response, this way, even if the file disappears, we will be fine